game/src/Credits/Credits.gd

```diff
@@ -0,0 +1,98 @@
+extends Control
+
+signal backButtonPressed
+
+###############
+# Credits CSV format
+# The project title row is the only requirement within the csv file, however
+# it can be on any row, so long as it exists.
+# ----------------------
+# title,project-title
+# role-name,person-name
+# role-name,person-name
+# role-name,person-name
+# ...
+###############
+
+@export var coreCreditsPath : String = 'res://common/credits.csv'
+
+# REQUIREMENTS:
+# * 1.5 Credits Menu
+# * SS-17
+
+# REQUIREMENTS
+# * FS-4
+func loadCreditFile(path):
+	var roles = {}
+	var coreCredits = FileAccess.open(path, FileAccess.READ)
+	
+	while not coreCredits.eof_reached():
+		var line = coreCredits.get_csv_line()
+		
+		# If the line does not have an identifiable role or is empty then skip it
+		if line[0] == '':
+			continue
+		
+		var role: String = line[0].to_upper()
+		var person: String = line[1]
+		
+		if role not in roles.keys():
+			roles[role] = []
+			roles[role].push_back(person)
+		else:
+			roles[role].push_back(person)
```

Comment:
```suggestion
	while not coreCredits.eof_reached():
		var line := coreCredits.get_csv_line()
		
		# If the line does not have an identifiable role or is empty then skip it
		if line[0].is_empty():
			continue
		
		var role := line[0].to_upper()
		var person := line[1]
		
		if role not in roles.keys():
			roles[role] = []
			roles[role].push_back(person)
		else:
			roles[role].push_back(person)
```
line, role, and person should be able to be inferred

Reply:
They can be inferred, I was just being explicit. Also I know is_empty is correct it was just giving me issues(I need to get better at comments sorry)